lib/rucio/core/rse_counter.py

```diff
@@ -138,6 +142,49 @@ def update_rse_counter(rse_id, session=None):
         update.delete(flush=False, session=session)
 
 
+@transactional_session
+def update_replica_counts(rse_id, session=None):
+    """
+    Read the replica counts and update rse_usage.
+
+    :param rse_id:   The rse_id to update.
+    :param session:  Database session in use.
+    """
+    none_value = None
+
+    # Get the counds of obsolete and deletably (expired) files
+    obsolete_files, obsolete_bytes = (session
+                                      .query(func.count(), func.sum(models.RSEFileAssociation.bytes))
```

Comment:
is there a way to make these blocks a bit better readable?

Reply:
Not sure what's unreadable about it. Using () to enclose multiple lines is now preferred and it clearly separates the clauses. Indentation is determined by PyCharm

Reply:
How about this:
```diff
+    obsolete_files, obsolete_bytes = (
+        session.query(func.count(), func.sum(models.RSEFileAssociation.bytes))
+               .with_hint(models.RSEFileAssociation, "INDEX_FFS(REPLICAS REPLICAS_TOMBSTONE_IDX)", 'oracle')
+               .filter(models.RSEFileAssociation.rse_id == rse_id,
+                       models.RSEFileAssociation.tombstone != null(),
+                       models.RSEFileAssociation.tombstone == OBSOLETE)
+               .one()
+    )
```

Reply:
👍 I prefer that one too. Both are PEP8 compatible I guess, but for me the lonely `session` looks strange. Bens version has the advantage of saving line-length space as well.